p/scripts/global_view.js

```diff
@@ -73,6 +73,16 @@ function init_close_panel() {
 			document.getElementById('overlay').classList.remove('visible');
 			return false;
 		};
+	document.addEventListener('keydown', ev => {
+		let k = (ev.key.trim() || ev.code).toUpperCase();
+		if (k === 'ESC') {
```

Comment:
That is, as opposed to:
```suggestion
		if (k === 'ESC' || k === 'ESCAPE') {
```
(assuming that any of that stuff is necessary *at all* given that IE11 is deprecated and it's probably for IE8 or something)